README.md

```diff
@@ -23,6 +24,19 @@ Edit `plugin.json` with your `id`, `name`, and `description`:
 }
 ```
 
+2. `go.mod` with your Go module path, following the `<hosting-site>/<repository>/<module>` convention:
+```
+module github.com/mattermost/mattermost-plugin-starter-template
```

Comment:
I'd say it would be a bit more clear if we don't use the same `github.com/mattermost/mattermost-plugin-starter-template` in the examples, and set it to an explicit example. Something in the lines of:
```suggestion
module github.com/example/my-plugin
```